volume/api.go

```diff
@@ -25,7 +25,19 @@ const (
 type Request struct {
 	Name    string
 	Options map[string]string `json:"Opts,omitempty"`
-	MountID string            `json:",omitempty"`
+	ID      string            `json:",omitempty"`
```

Comment:
Ok, today Request is the struct into which all commands are decoded into,
from docker. If ID is removed from Request, then I'll add actionHandler
types specific to mount and unmount and handler functions for mount and
unmount need to be made. I'll post an updated PR.

On Mon, Aug 1, 2016 at 7:13 PM, Brian Goff <notifications@github.com> wrote:

> In volume/api.go
> <https://github.com/docker/go-plugins-helpers/pull/56#discussion_r72979406>
> :
>
> > @@ -25,7 +25,19 @@ const (
> >  type Request struct {
> >  	Name    string
> >  	Options map[string]string `json:"Opts,omitempty"`
> > -	MountID string            `json:",omitempty"`
> > +	ID      string            `json:",omitempty"`
>
> Can remove this
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/go-plugins-helpers/pull/56/files/e988f25ae538a69697e4738ffde5f18b4e8ac84e#r72979406>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APHseFLidaAugU9KUeSjGtBX-EUORx45ks5qbff7gaJpZM4JVFvb>
> .
>